File: app/models/process_session.rb
class ProcessSession
    def self.start(uuid)
        ActionCable.server.broadcast uuid, {
            action: "start", msg: uuid
        }
        sleep(5)
        ActionCable.server.broadcast uuid, {
            action: "start", msg: uuid
        }
    end

    def self.upload_start(uuid, data)
        ActionCable.server.broadcast uuid, {
            action: "upload_start", msg: "DATA!!!#{data}"
        }
    end

    def self.processing(uuid, data)
        ActionCable.server.broadcast uuid, {
            action: "processing", msg: data
        }
    end
end

File: app/assets/javascripts/channels/process_session.js
App.process_session = App.cable.subscriptions.create("ProcessSessionChannel", {

    connected: function(data) {
    },
    received: function(data) {
        if (data.action === 'start'){
            $('#session_uuid').val(data.msg);
            return this.printMessage(data.msg);
        }
        else if (data.action === "upload_start") {
            return this.printMessage(data.msg);
        }

    },
    printMessage: function(message) {
        return $("#messages").append("<p>" + message + "</p>");
    }
});
